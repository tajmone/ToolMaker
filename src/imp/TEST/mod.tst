%%ESET VARIABEL(-6 MOD 4)
%%(VARIABEL)
%%((5-1) MOD (1+3))
%%(5-1 MOD 1+3)
%%BEGIN(modtest(num))
%%(num)
%%END(modtest)
%%EINCLUDE(modtest(3 MOD 6))

