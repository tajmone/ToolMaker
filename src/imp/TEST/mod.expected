########## mod ##########
-2
0
8
3

